pkg/apiserver/rest/usecase/authentication.go

```diff
@@ -288,6 +268,107 @@ func (a *authenticationUsecaseImpl) GetDexConfig(ctx context.Context) (*apisv1.D
 	}, nil
 }
 
+func (a *authenticationUsecaseImpl) UpdateDexConfig(ctx context.Context) error {
+	connectors, err := utils.GetDexConnectors(ctx, a.kubeClient)
+	if err != nil {
+		return err
+	}
+	dexConfigApp := &v1beta1.Application{}
+	if err := a.kubeClient.Get(ctx, types.NamespacedName{
+		Name:      keyDexConfig,
+		Namespace: velatypes.DefaultKubeVelaNS,
+	}, dexConfigApp); err != nil {
+		if kerrors.IsNotFound(err) {
```

Comment:
Where create this application first? if this app does not exist, we should ignore it.

Reply:
Dex addon will create this application